### PR TITLE
2.7.0 documentation update. 

### PR DESCRIPTION
**Pull request by badaltechiepi**
@keshavprasadms  Kindly review the doc and share your input.

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/data-pipeline.md [hunk 1]
```diff
@@ -24,25 +33,31 @@ Switch to the `Build` folder and run all jobs. For the value of the **github_rel
...
-4.Postgres 
-5.Yarn  
-6.Influxdb  
-7.Zookeeper   
-8.Kafka    
-9.Redis
+
+| Operation Name | Function              |
+| -------------- | --------------------- |
+| AnalyticsApi   | It install the dependecy to run the analytics API |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/data-pipeline.md]: It installs the dependency to run the analytics API

File: developer-docs/server-installation/data-pipeline.md [hunk 2]
```diff
@@ -24,25 +33,31 @@ Switch to the `Build` folder and run all jobs. For the value of the **github_rel
...
-4.DataProducts  
-5.Secor  
-6.KafkaIndexer  
-7.SamzaTelemertySchemas  
-8.Yarn (Multi-select all options in the job parameter job_names_to_deploy) 
+
+| Operation Name | Function              |
+| -------------- | --------------------- |
+| CassandraDbUpdate| It creates the Cassandra keyspace and do the db update|
+| KafkaSetup       | It create the Kafka topic |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/data-pipeline.md]: It creates the .......

File: faqs/test/test.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (NupurSancheti) [inline faqs/test/test.md]: please delete this file

File: developer-docs/server-installation/knowledge-platform.md [hunk 4]
```diff
@@ -11,7 +11,18 @@ This page explains the jobs to be run to bring up the Knowledge Platform service
 
 ## Build
 
-Switch to the `Build` folder and run all jobs. For the value of the **github_release_tag**, refer to [Current Release Tags and Jenkins Jobs Reference](developer-docs/server-installation/current_release_tags_n_jenkins_jobs){:target="_blank"}
+Switch to the `Build` folder and run all jobs in the below sequence. For the value of the **github_release_tag**, refer to [Current Release Tags and Jenkins Jobs Reference](developer-docs/server-installation/current_release_tags_n_jenkins_jobs){:target="_blank"}
+
+ Operation Name    | Function              |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/knowledge-platform.md]: formatting of the table to be corrected

File: developer-docs/server-installation/data-pipeline.md [hunk 5]
```diff
@@ -14,6 +14,15 @@ This page explains the jobs to be run to bring up the Data Pipeline services. In
 
 Switch to the `Build` folder and run all jobs. For the value of the **github_release_tag**, refer to [Current Release Tags and Jenkins Jobs Reference](developer-docs/server-installation/current_release_tags_n_jenkins_jobs){:target="_blank"}
 
+
+| Operation Name | Function              |
+| -------------- | --------------------- |
+| Analytics      | It generate the artifects for analytics api server|
+| ApiModule      |     |
```
Comment (keshavprasadms) [inline developer-docs/server-installation/data-pipeline.md]: Missing description

File: developer-docs/server-installation/data-pipeline.md [hunk 6]
```diff
@@ -14,6 +14,15 @@ This page explains the jobs to be run to bring up the Data Pipeline services. In
...
+| DataPipeline   | It generate the samza jobs artifects|
+| ETLJobs        | |
```
Comment (keshavprasadms) [inline developer-docs/server-installation/data-pipeline.md]: Missing description

File: developer-docs/upgrade/build_n_deploy.md [hunk 7]
```diff
@@ -32,18 +32,26 @@ The following is the list of jobs required to be built and deployed :
...
+|                          |                           |                    |                     |             | Keycloak                    |
+|                          |                           |                    |                     | KnowledgeMW | knowledge\-mw\-service      |
+|                          |                           |                    |                     |             | LogEsUpgrade6xx             |
+|                          |                           |                    |                     |             | oauth                       |
+|                          |
+|                          |                           |                    |                     |             | oauth                       |
+|                          |
+|                          |                           |                    |                     |             | Proxy                       |
+|                          |                           |                    |                     |             | OnboardAPIs                 |
```
Comment (keshavprasadms) [inline developer-docs/upgrade/build_n_deploy.md]: Many build jobs are missing, mappings to be corrected along with job names to match Jenkins job names exactly. Please work with @kali and get this table right.

Comment (kali) [reply developer-docs/upgrade/build_n_deploy.md]: Be aware, I think you're tagging the wrong "kali" person here :)

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by badaltechiepi** [comment]
@NupurSancheti Kindly view the doc

File: developer-docs/upgrade/build_n_deploy.md (hunk 7 above)
Comment (keshavprasadms) [reply developer-docs/upgrade/build_n_deploy.md]: Oops.. sorry

File: developer-docs/upgrade/jenkins_updates.md [hunk 8]
```diff
@@ -38,9 +38,9 @@ Follow the below steps to upgrade:
 
 5.The script will overwrite the existing jobs (jobs provided by default by Sunbird) and update the configurations with new changes of that release
 
-6.Go to Manage Jenkins → Configure System → Global Pipeline Libraries → Change Default Version to `release-2.6.0-shared-lib`
+6.Go to Manage Jenkins → Configure System → Global Pipeline Libraries → Change Default Version to `release-2.7.0-shared-lib`
 
-7.Go to Manage Jenkins → Configure System → Update `public_repo_branch` to `refs/tags/release-2.6.0` or go to Manage Jenkins → Configure System → Update `private_repo_branch` to the branch name of your private repo branch in case you created a new branch for release-2.6.0 inventory updates  
+7.Go to Manage Jenkins → Configure System → Update `public_repo_branch` to `refs/tags/release-2.7.0` or go to Manage Jenkins → Configure System → Update `private_repo_branch` to the branch name of your private repo branch in case you created a new branch for release-2.7.0 inventory updates  
 
 8.Restart your Jenkins and you are ready to go
```
Comment (NupurSancheti) [inline developer-docs/upgrade/jenkins_updates.md:45]: remove you are ready to go

File: developer-docs/upgrade/build_n_deploy.md [hunk 9]
```diff
@@ -32,18 +32,26 @@ The following is the list of jobs required to be built and deployed :
```
Comment (NupurSancheti) [inline developer-docs/upgrade/build_n_deploy.md:32]: no link on line no 29

File: developer-docs/upgrade/build_n_deploy.md [hunk 10]
```diff
@@ -32,18 +32,26 @@ The following is the list of jobs required to be built and deployed :
...
-|     Yarn                |          Yarn             |	    ApiModule      |  ApiModule          | Cassandra-migration | Cassandra-migration |
-|                         |    Neo4jDefinitionUpdate  |                    |  Secor              |            |    Keycloak |  
-|                         | 	                        |       Analytics    |   AnalyticsApi      |   Learner  |    Learner  |  
-|                         |                           |                    |   DataProducts      |    Lms     |    Lms      |  
-|  |   |  DataPipeline | Yarn (Multiselect all options in the job parameter job_names_to_deploy) | Content    |   Content   |
-|                         |                           |                    |                     | KnowledgeMW |  KnowledgeMW |  
-|                         |                           |	                   |                     |  Cert      |   Cert      |
-|                         |                           |                    |                     |  Player    |   Player    |  
+| Knowledge Platform Build | Knowledge Platform Deploy | DataPipeline Build | DataPipeline Deploy | Core Build  | Core Deploy                 |
```
Comment (NupurSancheti) [inline developer-docs/upgrade/build_n_deploy.md:35]: the table needs to be formatted again.. a lot of blank lines in the end

File: developer-docs/server-installation/core-services.md [hunk 11]
```diff
@@ -51,41 +51,40 @@ Ensure that all Artifacts are uploaded
...
 
 ### Deploy
 
-| Operation Name | Function |
-|--------------------|-----------| 
-| Adminutil | Deploys the Adminutil container | 
-| API Manager | Deploys the API manager Kong and API manager Echo | 
-| OnboardAPIS | Onboards all API's to Sunbird | 
-| OnboardConsumers 
+| Operation Name | Function | Description |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/core-services.md:64]: 2nd column is in present tense and 3 column is in the past tense.

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by badaltechiepi** [comment]
@NupurSancheti Kindly check the changes

File: developer-docs/server-installation/core-services.md [hunk 12]
```diff
@@ -51,41 +51,40 @@ Ensure that all Artifacts are uploaded
...
+| Cassandra | Does migration if required. Deploy this thrice by choosing different zip files using the build_number parameter. Ensure that you get a success message for the Cassandra migration on the Jenkins console output. Do not rely only on the red or green status indicator on Jenkins job | performs keyspace schema migration | 
+| CassandraTrigger | Deploys trigger jars for Cassandra |  Jars for Cassandra
+| (Provision) Keycloak | Provisions Keycloak by installing prerequisites like Java and environment variables |  install dependencies for keycloak | 
+| PlayerCDN | If you do not want to use CDN, run this job with Jenkins job parameter **cdn_enable** set to false (default). CDN  increases the performance of the web page and content for the end-user. Create CDN with the storage account and update the variable **sunbird_portal_cdn_url**. Set the Jenkins job parameter **cdn_enable** to true. This upload player static contents to CDN storage account. | uploads player CDN assets |
+| Player | deploys portal UI | UI for sunbird  |
+| Proxy | Deploys Proxy. Handles routing within the swarm | redirection and routing management |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/core-services.md:77]: redirects and routes

File: developer-docs/server-installation/core-services.md [hunk 13]
```diff
@@ -51,41 +51,40 @@ Ensure that all Artifacts are uploaded
...
+| OnboardConsumers | onboard sunbird consumers |onboard sunbird consumers |
+Update **core_vault_sunbird_api_auth_token**, **core_vault_kong__test_jwt** and **core_vault_sunbird_ekstep_api_key** with the **jwt token** from the Jenkins output of **api-management-test-user** if you are using the Knowledge Platform and Data Pipeline along with core| Generates user specific key |  Onboards new consumer to Sunbird and generates the consumer specific API key | 
+| (Provision) Cassandra | Provisions Cassandra and create keyspaces required for Sunbird Core | provisions Cassandra and creates keyspaces and performs migration |
+| Cassandra | Does migration if required. Deploy this thrice by choosing different zip files using the build_number parameter. Ensure that you get a success message for the Cassandra migration on the Jenkins console output. Do not rely only on the red or green status indicator on Jenkins job | performs keyspace schema migration | 
+| CassandraTrigger | Deploys trigger jars for Cassandra |  Jars for Cassandra
+| (Provision) Keycloak | Provisions Keycloak by installing prerequisites like Java and environment variables |  install dependencies for keycloak |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/core-services.md:74]: installs

File: developer-docs/server-installation/core-services.md [hunk 14]
```diff
@@ -51,41 +51,40 @@ Ensure that all Artifacts are uploaded
...
-|--------------------|-----------| 
-| (Deploy) ApplicationES | From the Deploy Folder, **Deploy ApplicationES** provisions for the Elasticsearch and creates indices necessay for Sunbird Core|
-| ESMapping (Under OpsAdministarion. Provide value as *all* for job parameter indices_name) | Creates Elasticsearch indexes |
-| Postgres | Provisions for Postgres |
-| PostgresDbUpdate | Creates the databases, assigns roles and creates users |
-| LogEsUpgrade6xx | Install elasticsearch 6.X |
+| Operation Name | Function | Description |
+|--------------------|-----------|---------|
+| (Deploy) ApplicationES | From the Deploy Folder, **Deploy ApplicationES** provisions for the Elasticsearch and creates indices necessary for Sunbird Core| Provisions Elasticsearch 6.2.3 version used by core services |
+| ESMapping (Under OpsAdministarion. Provide the value as *all* for job parameter indices_name) | Creates Elasticsearch indexes | Used to create the specific index with its mapping or all indices which will be used by our sunbird app |
```
Comment (NupurSancheti) [inline developer-docs/server-installation/core-services.md:57]: Creates the specific.... remove "used to" from all the instances

File: developer-docs/server-installation/data-pipeline.md [hunk 15]
```diff
@@ -24,25 +33,31 @@ Switch to the `Build` folder and run all jobs. For the value of the **github_rel
...
-3.AnalyticsApi  
-4.DataProducts  
-5.Secor  
-6.KafkaIndexer  
-7.SamzaTelemertySchemas  
-8.Yarn (Multi-select all options in the job parameter job_names_to_deploy) 
+
+| Operation Name | Function              | Description |
+| -------------- | --------------------- |-------------- |
+| CassandraDbUpdate| Updates the cassandra db.|It updates the required Cassandra table schema|
```
Comment (NupurSancheti) [inline developer-docs/server-installation/data-pipeline.md:56]: cassandra *DB* remove full stops from the end

File: developer-docs/server-installation/data-pipeline.md [hunk 16]
```diff
@@ -24,25 +33,31 @@ Switch to the `Build` folder and run all jobs. For the value of the **github_rel
...
-8.Kafka    
-9.Redis
+
+| Operation Name | Function              | Description|
+| -------------- | --------------------- |-------------|
+| AnalyticsApi   | Provision the server to run the analytics-apiserver|Creates the analytics user required folders and installs the dependencies|
+| AnalyticsSecor | Provision the server to run the secor jobs.|Creates the analytics user required folders and installs the dependencies to run secorjobs|
+| AnalyticsSpark | Provision the server to run the data products jobs| Creates the analyticsuser required folders and installs the dependencies to run spark|
+| Postgres       | Provision the Postgres database.| Installs the dependencies like java and install the Postgres database|
+| Yarn           | Provision the Yarn cluster |Install the dependencies required to create the yarn cluster and creates the yarn cluster to run samza jobs|
```
Comment (NupurSancheti) [inline developer-docs/server-installation/data-pipeline.md:43]: installs - in the following row as well

File: developer-docs/server-installation/jenkins-setup.md [hunk 17]
```diff
@@ -12,66 +12,64 @@ This page provides you with step-by-step instructions to set up the Jenkins serv
...
 > **Note** The options are case sensitive, the script displays the accepted options 
  
-13.After the script completes copying the job configurations, go to the browser and restart Jenkins using the command, `public-ip:8080/restart` OR `domain-name:8080/restart` 
+13. After the script completes copying the job configurations, go to the browser and restart Jenkins using the command, `public-ip:8080/restart` OR `domain-name:8080/restart` 
  
-14.Go to **http://(jenkins_domain)/credentials/store/system/domain/_/newCredentials** 
+14. Go to **http://(jenkins_domain)/credentials/store/system/domain/_/newCredentials** 
  
-15.Select **Username with Password** 
+15. Select **Username with Password** in Kind drop down and enter the username and password in the respected field of the GitHub account where the private repository is hosted
```
Comment (NupurSancheti) [inline developer-docs/server-installation/jenkins-setup.md:63]: point number 16 missing

File: developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md [hunk 18]
```diff
@@ -39,7 +39,7 @@ The following environment variables are used for Jenkins setup.
...
+|public_repo_branch|This is a unique variable. All the jobs in Jenkins are by default configured to checkout the Jenkinsfile from this variable as **${public_repo_branch}**. When a value like release-2.7.0 is provided to this variable, the Jenkins jobs checkout the Jenkins file from release-2.7.0 branch from the URL configured in the job. If this value is set to **refs/tags/release-2.7.0**, then Jenkins checks out the Jenkins file from the tag **release-2.7.0**. You can change this variable in Jenkins job configurations and specify a specific branch or tag name. This is useful when you want to run some jobs from a different branch or tag instead of the value mentioned in the variable.|
```
Comment (NupurSancheti) [inline developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md:42]: checkout the Jenkins file ??

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by badaltechiepi** [comment]
@keshavprasadms Marget the PR

File: developer-docs/server-installation/knowledge-platform.md [hunk 19]
```diff
@@ -23,35 +34,40 @@ For KnowledgePlatform build, use the default profile_id which is "platform_servi
...
-    8.Neo4jElasticSearchSyncTool   
-    9.KafkaSetup   
-    10.Yarn  
+ 
+| Operation Name      | Function              |
+| --------------      | --------------------- |
+| CassandraTrigger    | Deploys the CassandraTrigger jar file and installs the logstash |
+| CassandraDbUpdate   | Creates the Cassandra keyspace and do the Db update
+| Neo4j               | Deploys the Neo4j artefacts |confogures logstash and neo4j, and start logstash process
+| StartNeo4jCluster   | Starts the Neo4j | starts neo4j process
```
Comment (NupurSancheti) [inline developer-docs/server-installation/knowledge-platform.md:64]: | starts neo4j - To be removed

File: developer-docs/server-installation/knowledge-platform.md [hunk 20]
```diff
@@ -23,35 +34,40 @@ For KnowledgePlatform build, use the default profile_id which is "platform_servi
...
-    7.Neo4DefinitionUpdate  
-    8.Neo4jElasticSearchSyncTool   
-    9.KafkaSetup   
-    10.Yarn  
+ 
+| Operation Name      | Function              |
+| --------------      | --------------------- |
+| CassandraTrigger    | Deploys the CassandraTrigger jar file and installs the logstash |
+| CassandraDbUpdate   | Creates the Cassandra keyspace and do the Db update
+| Neo4j               | Deploys the Neo4j artefacts |confogures logstash and neo4j, and start logstash process
```
Comment (NupurSancheti) [inline developer-docs/server-installation/knowledge-platform.md:63]: |confogures to ", configures"

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/upgrade/build_n_deploy.md (hunk 9 above)
Comment (Sunitashelar) [reply developer-docs/upgrade/build_n_deploy.md:32]: done

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/faqs/test/test/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @badaltechiepi,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://574.qa.docs.sunbird.org/2.7.0/ .<br/> **Note:** Only _2.7.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://574.qa.docs.sunbird.org/2.7.0/developer-docs/developer-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/ansible-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/core-services/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/data-pipeline/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation// <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/plugins/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/jenkins_updates/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/developer-docs/upgrade/update_ansible_variables/ <br /> - http://574.qa.docs.sunbird.org/2.7.0/features-documentation/book_creation/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**